### ISSUE TITLE
1837 Phase names

### ISSUE THREAD

**Issue by erik-vos** [Rails-18xx/Rails#366]
From issue #362, reported by @jdgalt:

[8] In the menu Info -> Phases there are six Phase 2s and three Phase 3s.
(Actually this is ambiguous among the multiple rule sets out there -- the
current Lonny Orgler v2.0 uses phase number = train number. If we're not
going to do this then something should be done so the user can tell the
duplicates apart. It's also ambiguous within Rails -- the Report window
announces new phases as phase = train number.)

**Comment by erik-vos** [comment]
Yes, this is confusing. The phases reported come from the "realName" attributes per phase in `Game.xml`, and correspond to the v1 phase names. These "real names" are also used in the Game Status window, both for v1 and v2.
Actually, the v2 rules only mention phases 2, 3, 4 and 5. For simplicity, Rails defines a new separate phase each time buying a new train changes something.

I would suggest:
1. To have different "real names" for v1 and v2.
2. To have the Info display both the "train phase name" and the "real name" for either version, or perhaps all three: train, v1 and v2.
@neutronc, is this something for you to look at?

**Comment by neutronc** [comment]
we should go with train Number==Phase Name

**Comment by sbleeck** [comment]
this is now solved, 1837 is fully playable
